ansible/playbook.yml

```
@@ -0,0 +1,204 @@
+      become: yes
+
+    - name: Upgrade all packages to the latest version
+      apt:
+        name: "*"
+        state: latest
+        force_apt_get: yes
+      become: yes
+
+    - name: Install apache2
+      apt:
+        name: apache2
+      become: yes
+
+    - stat:
+        path: /var/www
+      register: sym_link
+
+    - name: Delete /var/www in case it is not symbolic link
+      file:
+        path: /var/www
+        state: absent
+      when: sym_link.stat.exists and not sym_link.stat.islnk
+      become: yes
+
+    - name: Create /var/www if necessary
+      file:
+        src: /vagrant
+        dest: /var/www
+        state: link
+      when: not sym_link.stat.exists or not sym_link.stat.islnk
+      become: yes
+
+    - name: Install apache2-dev
+      apt:
+        name: apache2-dev
+      become: yes
+
+    - name: Install tools
+      apt:
+        name:
+        - libpq-dev
+        - libc6-dev
+        - libjpeg62-dev
+        - libfreetype6-dev
+      become: yes
+
+    - name: Install the gpg key for nodejs LTS
+      apt_key:
+        url: "https://deb.nodesource.com/gpgkey/nodesource.gpg.key"
+        state: present
+      become: yes
+
+    - name: Install the nodejs LTS repos
+      apt_repository:
+        repo: "deb https://deb.nodesource.com/node_{{ NODEJS_VERSION }}.x {{ ansible_distribution_release }} main"
+        state: present
+        update_cache: yes
+      become: yes
+
+    - name: Install the nodejs
+      apt:
+        name: nodejs
+        state: present
+      become: yes
+      become_user: root
+
+    - name: Install the nodejs
+      apt:
+        name: npm
+        state: present
+      become: yes
+      become_user: root
+
+    - name: Instal yarn
+      npm:
+        name: yarn
+        global: yes
+      become: yes
+      become_user: root
+
+    - name: Install python3
+      apt:
+        name: python3
+      become: yes
+
+    - name: Install python3-venv
+      apt:
+        name: python3-venv
+      become: yes
+
+    - name: Install redis
+      apt:
+        name: redis-server
+      become: yes
+
+    - name: Add alias to .bash_alises
+      lineinfile:
+        path: ~/.bash_aliases
+        line: "alias run='python /vagrant/zapisy/run.py'"
+        regex: "^alias run='python /vagrant/zapisy/run.py'$"
+        state: present
+        insertafter: EOF
+        create: yes
+
+    - name: Add source command to .bashrc
+      lineinfile:
+        path: ~/.bashrc
+        line: "source ~/env3/bin/activate"
+        regex: "^source ~/env3/bin/activate$"
+        state: present
+        insertafter: EOF
+        create: yes
+
+    - name: Install pip
+      apt:
+        pkg:
+        - python-pip
+        - python3-venv
+        update_cache: yes
+      become: yes
+
+    - name: Upgrade pip
+      pip:
+        name: pip
+        state: latest
+      args:
+        chdir: ~/
+      become: yes
+
+    - name: Set up virtualenv
+      shell: python3.6 -m venv env3
```

```suggestion
      shell: python3 -m venv env3
```
Dzięki temu nie trzeba będzie wszystkiego poprawiać, gdy przejdziemy na nowe (niedawno wydane) Ubuntu i Pythona 3.8.